### PR TITLE
[DI2-197] Replace Redux Router links to home with React Router links

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

File: src/app/pages/ContentPage.jsx [hunk 1]
```diff
@@ -19,7 +21,12 @@ const ContentPage = ({ title, children }) => (
           {children}
           <Paragraph>
             Of ga door naar de{' '}
-            <Link variant="inline" href="/" title="Naar Data en Informatie - Homepage">
+            <Link
+              as={RouterLink}
+              to={toHome()}
+              variant="inline"
+              title="Naar Data en Informatie - Homepage"
```
Comment (janjaap) [inline src/app/pages/ContentPage.jsx:28]: `title` Attribute is hardly ever necessary. See https://www.a11yproject.com/posts/2013-04-22-title-attributes/#:~:text=HTML%20title%20attributes%20are%20often,and%20possibly%20not%20even%20used. for reference

File: src/panorama/containers/PanoramaContainer.jsx [hunk 2]
```diff
@@ -88,6 +90,29 @@ class PanoramaContainer extends Component {
...
+    } else {
+      // eslint-disable-next-line react/destructuring-assignment
+      this.props.toGeoSearch({
+        [PARAMETERS.LOCATION]: panoramaLocation,
+        [PARAMETERS.VIEW]: ViewMode.Split,
+        [PARAMETERS.LAYERS]: overlays,
+      })
+    }
+  }
+
```
Comment (janjaap) [inline src/panorama/containers/PanoramaContainer.jsx:115]: Tests have been removed from the saga. Can you reintroduce them here?

File: src/store/middleware/matomo/trackEvents.js [hunk 3]
```diff
@@ -98,8 +97,6 @@ const trackEvents = {
 
     return []
   },
-  // NAVIGATION -> CLICK CLOSE FROM PANORAMA
-  [CLOSE_PANORAMA]: () => [MATOMO_CONSTANTS.TRACK_EVENT, 'navigation', 'panorama-verlaten', null],
```
Comment (janjaap) [inline src/store/middleware/matomo/trackEvents.js:102]: This action shouldn't be tracked anymore?

Comment (jonkoops) [reply src/store/middleware/matomo/trackEvents.js:102]: Yeah it should, I am looking at how we can make calls to matomo from a a class component.

File: src/panorama/containers/PanoramaContainer.jsx (hunk 2 above)
Comment (jonkoops) [reply src/panorama/containers/PanoramaContainer.jsx:115]: Sure, will do.

File: src/app/pages/ContentPage.jsx (hunk 1 above)
Comment (jonkoops) [reply src/app/pages/ContentPage.jsx:28]: This is a prop that will be used in the component, but not as the title attribute.

File: src/store/middleware/matomo/trackEvents.js (hunk 3 above)
Comment (jonkoops) [reply src/store/middleware/matomo/trackEvents.js:102]: Done.

File: src/panorama/containers/PanoramaContainer.jsx (hunk 2 above)
Comment (jonkoops) [reply src/panorama/containers/PanoramaContainer.jsx]: Took a look at the tests removed from the saga and they are not really testing anything. One was skipped and the other was testing if the correct action was dispatched from the saga, neither of which is relevant to the new situation.

**Review by chrisvanmook — APPROVED** [review]

File: src/panorama/ducks/actions.js [hunk 4]
```diff
@@ -54,12 +53,6 @@ export const fetchPanoramaError = (error) => ({
   payload: error,
 })
 
-export const closePanorama = () => ({
-  type: CLOSE_PANORAMA,
-  meta: {
-    tracking: true,
```
[the remark this answers is not in the document]
Comment (jonkoops) [reply src/panorama/ducks/actions.js:60]: Jup

File: src/panorama/containers/PanoramaContainer.jsx (hunk 2 above)
Comment (jonkoops) [reply src/panorama/containers/PanoramaContainer.jsx]: I've added tests to cover all the cases that occur when closing the panorama.

**Review by janjaap — APPROVED** [review]

**Pull request merged**